pkg/asset/manifests/cloudproviderconfig.go

```diff
@@ -92,6 +94,24 @@ func (cpc *CloudProviderConfig) Generate(dependencies asset.Parents) error {
 		}
 
 		cm.Data[cloudProviderConfigDataKey] = openstackmanifests.CloudProviderConfig(cloud.CloudConfig)
+
+		// Get the ca-cert-bundle key if there is a value for cacert in clouds.yaml
+		caFile := []byte{}
+		caFilePath := cloud.CloudConfig.CACertFile
+		if caFilePath != "" {
+			_, err := os.Stat(caFilePath)
+			if os.IsNotExist(err) {
+				return errors.Wrap(err, "cacert from clouds.yaml file path does not exist")
+			} else if err != nil {
+				return errors.Wrap(err, "could not read cacert from clouds.yaml")
+			}
+
+			caFile, err = ioutil.ReadFile(caFilePath)
+			if err != nil {
+				return errors.Wrap(err, "could not read cacert from clouds.yaml")
+			}
+		}
+		cm.Data["ca-bundle.pem"] = string(caFile)
```

Comment:
"ca-bundle.pem", "ca-cert.pem"... I'll let you pick a name and use the same name in the configmap key and filename of the well-known location where the cert is mounted.